packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -16,33 +16,142 @@ const IconArrow = ({ className, classNamePath }) => (
     </svg>
 );
 
+const KEY_CODE = {
+    ENTER  : 13,
+    ESCAPE : 27,
+    TAB    : 9,
+    KEYDOWN: 40,
+    KEYUP  : 38,
+};
+
+const not_found_default_text = 'No results found';
```

Comment:
Do we need to localize this?

Reply:
I think common components should always receive the localized text from outside?  I added the prop not_found_text this is a default in case that prop is not passed.

Reply:
using a defaultprop instead of this const